### PR TITLE
Business site - Andrea

### PR DESCRIPTION
**Pull request by Andrealarsson**
Here is my business site for my own  travel agency.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,117 @@
...
+        <div class="submit">
+          <button type="submit">Login</button>
+        </div>
+      </form>
+    </div>
+
+    <!-- Accordion -->
+    <div class="container">
+      <h2>Q&A</h2>
+      <div class="accordion">
```
Comment (EmOlsz) [inline code/index.html:87]: Well designed structure - good job :)

File: code/script.js [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+//JavaScript for the Accordion
+
+const accordionHeader1 = document.getElementById("accordion-header-1");
+const accordionBody1 = document.getElementById("accordion-body-1");
+
+
+
```
Comment (EmOlsz) [inline code/script.js:7]: We don't need to add so many empty lines - one is completely enough. Let's fix that

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,233 @@
...
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  background-color: rgba(58, 116, 114, 0.24);
+}
+
+/*NAVBAR*/
+ul {
```
Comment (EmOlsz) [inline code/style.css:8]: Remember - we shouldn't style by anything else than CSS classes. Let's add classes in HTML and style them over here

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,233 @@
...
+
+button {
+  padding: 10px 20px;
+  background-color: rgba(255, 255, 255, 0);
+  border: 2px solid rgb(58, 116, 114);
+  border-radius: 10px;
+  color: rgb(26, 53, 52);
+  font-size: 18px;
+  margin-top: 20px;
+  cursor: pointer;
```
Comment (EmOlsz) [inline code/style.css:139]: Great you remembered about `pointer` value!

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Great job Andrea!
Your project is well structured and consequent. There are only some small things to fix - take a look at my comments and provide changes.
Moreover, take a look at the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475)

**Comment by Andrealarsson** [comment]
Changes done! :)

**Comment by EmOlsz** [comment]
Thank you, project approved :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**